### PR TITLE
[Skrifa] Implement COLRv0 traversal

### PR DESCRIPTION
**Pull request by drott**
Now that COLRv1 is in place, COLRv0 is a relatively trivial addition, sending a sequence of push clip, fill solid, pop clip operations. Add the implementation and a test based on the COLRv0 glyph in the test font.

Briefly tested in Skia using https://github.com/googlefonts/color-fonts/blob/main/glyph_descriptions.md#:~:text=%F3%B0%B8%80-,U%2BF0E00,-colored_circles_v0

### CODE REVIEW

**Review by dfrg — APPROVED** [review]
Nice! Now that this is complete, I'm hoping to find some time over the holidays to test it out with some other rendering backends.

File: skrifa/src/color/traversal.rs [hunk 1]
```diff
@@ -447,6 +447,24 @@ pub(crate) fn traverse_with_callbacks(
...
 }
 
+pub(crate) fn traverse_v0_range(
+    range: &Range<usize>,
+    instance: &ColrInstance,
+    painter: &mut impl ColorPainter,
+) -> Result<(), PaintError> {
+    for layer_index in range.clone() {
+        let (layer_index, palette_index) = (*instance).v0_layer(layer_index)?;
+        // TODO(drott): Use optimized callback function combining clip, fill and transforms.
```
Comment (rsheeter) [inline skrifa/src/color/traversal.rs:457]: Request: file issues rather than TODO's, it's far more visible.

Comment (rsheeter) [reply skrifa/src/color/traversal.rs:457]: (unless you plan to address basically immediately)

**Comment by rsheeter** [comment]
> COLRv0 is a relatively trivial addition

Fantastic to see!

File: skrifa/src/color/traversal.rs (hunk 1 above)
Comment (drott) [reply skrifa/src/color/traversal.rs]: TODO linking to newly filed issue #746 now.

**Pull request merged**